pkg/kubecost/allocation.go

```diff
@@ -75,7 +75,6 @@ type Allocation struct {
 	RAMCost                    float64               `json:"ramCost"`
 	RAMCostAdjustment          float64               `json:"ramCostAdjustment"`
 	SharedCost                 float64               `json:"sharedCost"`
-	SharedCostAdjustment       float64               `json:"sharedCostAdjustment"`
```

Comment:
Why is this being removed?

Reply:
@Sean-Holcomb Do we want this field or not? I removed because I thought we had discussed not needing a `SharedCostAdjustment` field, and considered it possibly mistake from merge, but perhaps I am mistaken.

Reply:
Yes that should be removed, it is an unused field as Niko says we decided to move the values elsewhere